Documentation/openid-connect.md

```diff
@@ -1,10 +1,12 @@
 # An overview of OpenID Connect
 
-This document attempts to provide an overview of the [OpenID Connect protocol](
+This document attempts to provide a general overview of the [OpenID Connect protocol](
 https://openid.net/connect/), a flavor of OAuth2 that dex implements. While
 this document isn't complete we hope it provides a enough to get users up and
 running.
 
+For an overview of custom claims, scopes, and client features implemented by dex, see [this document][scopes-claims-clients].
+
 ## OAuth2
 
 OAuth2 should be familiar to anyone who's used something similar to a "Login
```

Comment:
Since this doc is only tangentially part of this PR, going to hold off on that change.